src/rgw/rgw_sal.h

```diff
@@ -791,7 +791,7 @@ class Bucket {
     /* dang - This is temporary, until the API is completed */
     virtual rgw_bucket& get_key() = 0;
     virtual RGWBucketInfo& get_info() = 0;
-
+    
```

Comment:
This tab addition should still be removed from this PR

Reply:
done!